test/distributed/org/apache/cassandra/distributed/test/RepairErrorsTest.java

```diff
@@ -92,8 +92,7 @@ public void testRemoteSyncFailure() throws Exception
         try (Cluster cluster = init(Cluster.build(3)
                                            .withConfig(config -> config.with(GOSSIP)
                                                                        .with(NETWORK)
-                                                                       .set("disk_failure_policy", "stop")
-                                                                       .set("disk_access_mode", "mmap_index_only"))
```

Comment:
Even though the default has now changed, this continues to be a necessary condition for the test, and leaving it place to document that seems like a good idea. WDYT?

Reply:
ok I ll revert this change. Thanks @maedhroz